pkg/cri/sbserver/container_create.go

```diff
@@ -1018,6 +1042,16 @@ func (c *criService) linuxContainerMounts(sandboxID string, config *runtime.Cont
 			HostPath:       sandboxDevShm,
 			Readonly:       false,
 			SelinuxRelabel: sandboxDevShm != devShm,
+			// XXX: tmpfs support for idmap mounts got merged in
+			// Linux 6.3.
+			// Our CI runs with 5.15 kernels, so disabling idmap
```

Comment:
```suggestion
			// Our Ubuntu 22.04 CI runs with 5.15 kernels, so disabling idmap
```
We also have Fedora 38 CI which currently uses kernel 6.2.
In the next month we will get Fedora 39 with a newer kernel.

Reply:
Good point, fixed!